### PR TITLE
265873 show tls cert status

### PR DESCRIPTION
**Pull request by nilsgstrabo**

### CODE REVIEW

File: src/components/external-dns-alias-help/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Icon, Typography } from '@equinor/eds-core-react';
+import { info_circle } from '@equinor/eds-icons';
+
+import externalUrls from '../../externalUrls';
```
Comment (Pespiri) [inline src/components/external-dns-alias-help/index.tsx:4]: use named import
```suggestion
import { externalUrls } from '../../externalUrls';
```

File: src/components/secret-form/index.tsx [hunk 2]
```diff
@@ -9,7 +9,12 @@ import { ChangeEvent, ReactNode, useEffect, useState } from 'react';
 
 import { Alert } from '../alert';
 import { SecretStatus } from '../secret-status';
+import { SecretStatusMessages } from '../secret-status-messages';
+import { TLSCertificateList } from '../tls-certificate-list';
+import { ExternalDnsAliasHelp } from '../external-dns-alias-help';
 import { SecretModel, SecretModelValidationMap } from '../../models/secret';
+import { SecretType } from '../../models/secret-type';
+import { SecretStatus as SecretStatusEnum } from '../../models/secret-status';
```
Comment (Pespiri) [inline src/components/secret-form/index.tsx:17]: remove unused import
```suggestion
```

File: src/components/secret-status-messages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import { SecretStatus } from '../../models/secret-status';
+
+const AlertTemplates: Record<SecretStatus, Omit<AlertProps, 'children'>> = {
+  [SecretStatus.Pending]: { type: 'info' },
+  [SecretStatus.NotAvailable]: { type: 'warning' },
+  [SecretStatus.Invalid]: { type: 'danger' },
+  [SecretStatus.Consistent]: { type: 'info' },
+  [SecretStatus.Unsupported]: { type: 'danger' },
+};
+export const SecretStatusMessages = ({
```
Comment (Pespiri) [inline src/components/secret-status-messages/index.tsx:14]: add spacing
```suggestion
};

export const SecretStatusMessages = ({
```

File: src/components/secret-status-messages/index.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { Typography } from '@equinor/eds-core-react';
+
+import Alert, { AlertProps } from '../alert';
```
Comment (Pespiri) [inline src/components/secret-status-messages/index.tsx:3]: used named imports
```suggestion
import { Alert, AlertProps } from '../alert';
```

File: src/components/status-badges/component-secret-status-badge.tsx [hunk 5]
```diff
@@ -9,24 +9,34 @@ import {
 
 import { SecretStatus } from '../../models/secret-status';
 
+export type ComponentSecretStatusBadgeProps = {
+  status: SecretStatus;
+};
```
Comment (Pespiri) [inline src/components/status-badges/component-secret-status-badge.tsx:14]: remember to change the PropTypes.ValidationMap typing as well
`PropTypes.ValidationMap<ComponentSecretStatusBadgeProps>`

```suggestion
export interface ComponentSecretStatusBadgeProps {
  status: SecretStatus;
}
```

File: src/models/secret/test-data.ts [hunk 6]
```diff
@@ -13,11 +15,14 @@ export const testData: TestDependencyDataType<SecretModel> = [
   {
     __testDescription: 'Valid full object',
     name: 'name',
+    id: 'id',
```
Comment (Pespiri) [inline src/models/secret/test-data.ts:18]: remember to also update testdata for 'Invalid empty object', just add the same fields and data as you did for 'Valid full object'

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: src/models/secret/test-data.ts (hunk 6 above)
Comment (nilsgstrabo) [reply src/models/secret/test-data.ts:18]: updated

**Review by Pespiri — APPROVED** [review]

**Pull request merged**